media/css/firefox/family/_utils.scss

```diff
@@ -38,8 +38,16 @@ $border-width: 3px;
 }
 
 // Shadows
+:root {
+    --shadow-offset: #{$spacing-sm};
+    @media #{$mq-md} {
+        --shadow-offset: 14px;
+    }
+}
+
```

Comment:
**Question**: Wondering why we decided to use a css variable here?

Reply:
Oh, good question. I wanted to isolate the variable change in media queries and [sass variables don't work this way](https://github.com/sass/sass/issues/1639#issuecomment-74273150), so I switched to css variables.